src/SipConnector.ts

```diff
@@ -937,21 +951,35 @@ export default class SipConnector {
     });
   };
 
-  _generateStream(videoTrack, audioTrack) {
-    const { id } = videoTrack;
+  _generateStream({
+    videoTrack,
+    audioTrack,
+  }: {
+    videoTrack?: MediaStreamTrack;
+    audioTrack?: MediaStreamTrack;
```

Comment:
in order to get away from unnecessary conditions, you can create a separate method for this

Reply:
New method for generating audio stream has added